language/tools/move-unit-test/src/lib.rs

```diff
@@ -117,10 +126,14 @@ impl UnitTestingConfig {
             test_runner.filter(filter_str)
         }
 
-        test_runner
-            .run(&shared_writer)
-            .unwrap()
-            .summarize(&shared_writer)?;
+        let test_results = test_runner.run(&shared_writer).unwrap();
+
+        if self.report_statistics {
+            test_results.report_statistics(&shared_writer)?;
+        }
+
```

Comment:
Is there any reason why we have the empty line here and in line 130?

Reply:
Not really, just style. I'll remove :)